zoo/board_games/tictactoe/config/tictactoe_gumbel_muzero_bot_mode_config.py

```diff
@@ -0,0 +1,90 @@
+from easydict import EasyDict
+
+# ==============================================================
+# begin of the most frequently changed config specified by the user
+# ==============================================================
+collector_env_num = 8
+n_episode = 8
+evaluator_env_num = 5
+num_simulations = 30
+update_per_collect = 50
+batch_size = 256
+max_env_step = int(2e5)
+reanalyze_ratio = 0.
+# ==============================================================
+# end of the most frequently changed config specified by the user
+# ==============================================================
+
+tictactoe_gumbel_muzero_config = dict(
+    exp_name=
+    f'data_mz_ctree/tictactoe_gumbel_muzero_bot-mode_ns{num_simulations}_upc{update_per_collect}_rr{reanalyze_ratio}_seed0',
+    env=dict(
+        battle_mode='play_with_bot_mode',
+        collector_env_num=collector_env_num,
+        evaluator_env_num=evaluator_env_num,
+        n_evaluator_episode=evaluator_env_num,
+        manager=dict(shared_memory=False, ),
+    ),
+    policy=dict(
```

Comment:
can we simplify some fields with its default value

Reply:
all the configs that are not frequently changed here differ from the default config